src/cli/argv_parser.ts

```diff
@@ -2,17 +2,29 @@ import _ from 'lodash'
 import { Command } from 'commander'
 import path from 'path'
 import Gherkin from 'gherkin'
+import { SnippetInterface } from '../formatter/step_definition_snippet_builder/snippet_syntax'
 
 // Using require instead of import so compiled typescript will have the desired folder structure
 const { version } = require('../../package.json') // eslint-disable-line @typescript-eslint/no-var-requires
 
+export interface IParsedArgvFormatRerunOptions {
+  separator?: string
+}
+
+export interface IParsedArgvFormatOptions {
```

Comment:
As we support arbitrary format options for custom formatters, I think we need to extend this interface to allow that, something like:

```typescript
export interface IParsedArgvFormatOptions {
  colorsEnabled?: boolean
  rerun?: IParsedArgvFormatRerunOptions
  snippetInterface?: SnippetInterface
  snippetSyntax?: string
  [customKey: string]: any
}
```

Could look at using [tsd](https://www.npmjs.com/package/tsd) to put in guards for stuff like this.

Reply:
Added `[customKey: string]: any`